lib/segment/src/id_tracker/compressed/compressed_point_mappings.rs

```diff
@@ -30,9 +30,10 @@ pub type FileEndianess = LittleEndian;
 
 #[derive(Clone, PartialEq, Default, Debug)]
 pub struct CompressedPointMappings {
-    // `deleted` specifies which points of internal_to_external was deleted.
-    // It is possible that `deleted` can be longer or shorter than `internal_to_external`.
-    // - if `deleted` is longer, then extra bits should be set to `false` and ignored.
+    /// `deleted` specifies which points of internal_to_external was deleted.
+    /// It is possible that `deleted` can be longer or shorter than `internal_to_external`.
+    /// - if `deleted` is longer, then extra bits should be set to `false` and ignored.
+    /// - if `deleted` is shorter, then extra indices are as if the bits were set to `true`.
```

Comment:
This is incorrect. In this structure the deleted flags are exactly sized. See [this](https://github.com/qdrant/qdrant/blob/c8898a19a4d63480543d5f05bbe33b4ddd35f716/lib/segment/src/id_tracker/compressed/compressed_point_mappings.rs#L50-L52) a bit further down.

I understand the comment was already there, but I only notice it now so let's change it here.

```suggestion
    /// `deleted` specifies which points of internal_to_external was deleted.
    /// Its size is exactly the same as `internal_to_external`.
```